### ISSUE TITLE
Don't use the project structure as default writing location

### ISSUE THREAD

**Issue by refs** [owncloud/ocis#113]
Currently konnectd's default writes a private key and a self signed certificate to the binary running location, leaving the programmer to either exclude this files from the git tree or in general adding some noise. If this is intended only for development [we can use a tmp dir](https://golang.org/pkg/io/ioutil/#TempDir)

![image](https://user-images.githubusercontent.com/6905948/74651075-56613c00-5183-11ea-9c67-771a329f96b3.png)

**Comment by refs** [comment]
cc @IljaN @butonic

**Comment by IljaN** [comment]
Yes the current implementation is not ideal. For the certs temp-dir would be okay :+1:  

But we are also creating [default config-files](https://github.com/owncloud/ocis-konnectd/blob/master/pkg/service/v0/service.go#L111). Need to do more concepting. A better approach would be to commit required default-configs to the configs dir. But this would break the current zero-configuration approach i.e just run the binary.  

A counter-point would be that the binary will always be delivered in a tar anyways so we can also ship the config dir with it.
